packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+
+##### Request Example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}"
+```
+
+##### Response Types
+
+HTTP 204:
+*No Content*
+
+### Get a rule
+
+<ApiOperation method="get" url="/api/v1/policies/${rulesetId}/rules/${ruleId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+The rule ID described in the [Rule object](#rule-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rules object](#rules-object)
+
+### Update a rule
+
+<ApiOperation method="put" url="/api/v1/policies/${rulesetId}/rules/${ruleId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+The rule ID described in the [Rule object](#rule-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+    "type": "Okta:SignOn",
+    "name": "Example Updated Rule",
+    "status": "ACTIVE",
+    "default": false,
+    "priority": 0,
+    "conditions": [
+       {
+          "key": "Okta:User",
+          "op": "IN_LIST",
+          "value": [
+             "00u1pdsiceJZLRJMSEUA"
+          ]
+       }
+    ],
+    "action": "ALLOW",
+    "requirement: [
+       "verificationMethod": {
+           "type": "ASSURANCE",
+           "factorMode": "2FA",
+           "constraints": [
+              {
+                "knowledge": {
+                   "types": [
+                     "password"
+                   ]
+                },
+                "possession": {
+                   "methods": [ "PUSH" ]
+                }
+              }
+           ],
+           "reauthenticateIn": "PT1M"
+       }
+    ]
+}' "https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rule object](#rule-object)
+
+### Activate A Rule
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/rules/${ruleId}/lifecycle/activate" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#ruleset-object) is required.
+
+The rule ID described in the [Rule object](#rule-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}/lifecycle/activate"
+```
+
+##### Response Types
+
+HTTP 204:
+*No content*
```

Comment:
I assume this should have the same note as Activate a Rule Set: "No Content is returned when the activation is successful"?

Reply:
Correct